packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -1,13 +1,15 @@
 import { EpochNo } from '../Block';
 import { Lovelace } from '../Value';
+import { OpaqueNumber } from '../../util';
 import { PoolIdHex } from './primitives';
 import { PoolParameters } from './PoolParameters';
 import { TransactionId } from '../Transaction';
 
 /**
  * Within range [0; 1]
  */
-export type Percent = number;
+export type Percent = OpaqueNumber<'Percent'>;
+export const Percent = (value: number): Percent => value as unknown as Percent;
```

Comment:
Would be nice to validate `value` range here

Reply:
Do you remember which tests? I guess APY could be >100% in local network :thinking: Anyway, please update the doc comment then